flatland/envs/rail_env.py

```diff
@@ -79,12 +81,15 @@ def __init__(self,
                  line_generator: "LineGenerator" = None,
                  number_of_agents=2,
                  obs_builder_object: ObservationBuilder = GlobalObsForRailEnv(),
-                 malfunction_generator_and_process_data=None,  # mal_gen.no_malfunction_generator(),
-                 malfunction_generator=None,
+                 malfunction_generator_and_process_data=None,
+                 malfunction_generator: "MalfunctionGenerator" = None,
                  remove_agents_at_target=True,
                  random_seed=None,
                  record_steps=False,
                  timetable_generator=ttg.timetable_generator,
+                 acceleration_delta=1.0,
+                 braking_delta=-1.0,
```

Comment:
```suggestion
                 breaking_delta=-1.0,
```
same in other places